packages/app/src/components/pages/Home/NewFilesToReview.js

```diff
@@ -38,21 +61,44 @@ const NewFilesToReview = () => {
       </thead>
       <tbody>
         <tr className="home-new-files-headers">
-          <td className="add-padding-left"><h2><strong>Application Name</strong></h2></td>
-          <td><h2><strong>HP Name</strong></h2></td>
-          <td><h2><strong>Family Member Name</strong></h2></td>
-          <td><h2><strong>Received Date</strong></h2></td>
-          <td><h2><strong>Stage</strong></h2></td>
+          <td className="add-padding-left">
+            <h2 onClick={() => requestSort('nominationName')} style={{cursor: 'pointer'}}>
+              <strong>Application Name {renderSortArrow('nominationName')}</strong>
+            </h2>
+          </td>
+          <td>
+            <h2 onClick={() => requestSort('providerName')} style={{cursor: 'pointer'}}>
+              <strong>HP Name {renderSortArrow('providerName')}</strong>
+            </h2>
+          </td>
+          <td>
+            <h2 onClick={() => requestSort('representativeName')} style={{cursor: 'pointer'}}>
+              <strong>Family Member Name {renderSortArrow('representativeName')}</strong>
+            </h2>
+          </td>
+          <td>
+            <h2 onClick={() => requestSort('dateReceived')} style={{cursor: 'pointer'}}>
+              <strong>Received Date {renderSortArrow('dateReceived')}</strong>
+            </h2>
+          </td>
```

Comment:
Same comment here as above. I think we could clean this up a bit by rendering these sortable headers within a helper function.